Lesson4/Task2/Editor.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+
+namespace Task2
+{
+    public class Editor
+    {
+        private AbstractHandler _handler;
```

Comment:
Constructor should create ready to use object. Now if you use Editor without call to ChooseDocument, it will throw an exception. You can create constructor with filePath parater to init this filed.